src/components/BusinessResultCard/BusinessResultCard.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { withStyles } from '@material-ui/core/styles';
+
+import {
+  Card, CardContent, Checkbox, Typography,
+} from '@material-ui/core';
+
+import { businessProps } from '../../types';
+
+const styles = (theme) => ({
+  card: {
+    maxWidth: 345,
+    maxHeight: 160,
+    borderColor: theme.palette.primary.main,
+  },
+  checked: {
+    backgroundColor: `${theme.palette.primary.light}20`,
+  },
+  title: {
+    display: 'inline',
+  },
+  checkbox: {
+    float: 'right',
+    padding: 0,
+    borderColor: theme.palette.primary.main,
+  },
+  wrapper: {
+    display: 'flex',
+    '& h6': {
+      fontWeight: 600,
+      marginRight: 15,
+    },
+  },
+});
+
+const BusinessResultCard = ({
```

Comment:
This is to display the results from yelp. 

![image](https://user-images.githubusercontent.com/1613526/103181285-9433f400-486d-11eb-92e1-6ed2a4926706.png)